### ISSUE TITLE
Issues regarding reproducibility

### ISSUE THREAD

**Issue by ashtsang** [bowang-lab/BLEEP#2]
Hi,
First of all, thank you for sharing this project. I am trying to reproduce the results in the paper but am running into multiple issues.
1) How do I reproduce the data processing step to generate the instances used during training? How do you retrieve batch corrected expression data from Harmony (since Harmony batch correction is done in the latent space)?
2) Can you give the set of instructions and hyper-parameters used to reproduce training and obtain the reported performances?
3) Can you release the model checkpoint and embeddings that were used to generate the reported numbers along with instructions for evaluation?

Thank you!

**Comment by Xrioen** [comment]
Hi, thanks for the questions. 
1. Here I used the normalized and log1p transformed expression and used the highly variable genes as the latent dimensions. After correction the expression patterns of the genes are well conserved although I do agree perhaps a non-latent approach could be better conceptually. However I suspect other batch correction methods will likely achieve similar results in practice. Furthermore, I've since tested models trained on the normalized + log transformed hvg matrices and achieved comparable (slightly worse) results. So for datasets that have relatively low amount of batch effect (like the dataset I've used) we can get away with no batch correction. 
2. Please find the hyperparameters in the supplementary of the paper, and also in the config.py file
3. I hesitate to release the model checkpoint since this paper is more of a proof of concept with the model being trained exclusively on human liver visium data, and likely won't generalize to other tissues or modalities. I'm currently working on a more generalizable model which I will be happy to release once completed!
